src/containers/Viewer/Viewer.tsx

```diff
@@ -288,27 +289,39 @@ const Viewer = ({ onClose }: ViewerProps) => {
         alt={selectedReviewable.label || selectedReviewable.filename}
       />
     )
-  } else if (!isFetchingReviewables) {
+  } else if (!isFetchingReviewables && versionReviewableIds?.length === 0) {
     let message = 'No preview available'
-    let children = null
 
     if (noVersions) {
       message = 'This task has published no versions.'
```

Comment:
If there are no versions then there is nothing to upload reviewables to so we the upload should be hidden.

Same goes for ''File not supported and needs conversion'.

![image](https://github.com/user-attachments/assets/87a43458-055f-452e-b495-a6a62a239656)